videoframes.py

```diff
@@ -79,6 +79,10 @@ def mouseReleaseEvent(self, event):
             else:
                 self.toggle_playback()
 
+    def mouseDoubleClickEvent(self, event):
+        """Toggles fullscreen mode on the current stream."""
+        self._fullscreen(self)
```

Comment:
We should probably check if no modifer keys are pressed while fullscreening. Currently if one holds ctrl (the modifier key for switching location of frames) the frame is selected, yet it is still fullscreened by dubble pressing.